eox_core/api/v1/views.py

```diff
@@ -22,7 +22,7 @@
     EdxappCourseEnrollmentQuerySerializer,
     EdxappUserReadOnlySerializer
 )
-from eox_core.edxapp_wrapper.users import create_edxapp_user, get_edxapp_user
+from eox_core.edxapp_wrapper.users import create_edxapp_user, get_edxapp_user, update_edxapp_user
```

Comment:
Please apply the trailing comma here also @anmrdz.

Reply:
never mind, since it's the same line of code. Im approving @anmrdz

Reply:
Done.